dev-lang/ispc/ispc-1.24.0-r1.ebuild

```diff
@@ -21,7 +21,14 @@ SRC_URI="
 LICENSE="BSD BSD-2 UoI-NCSA"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-IUSE="examples gpu openmp test"
+ISPC_LLVM_TARGETS=(
+	ARM # we force this on arm
+	WebAssembly # TODO needs dev-util/emscripten
+	X86 # we force this on x86
```

Comment:
I'm confused. You're adding these targets to ispc but you aren't depending on their presence in LLVM?

Reply:
Wasn't there prior so never thought to add it. Fixed.